Create ui-c8y-1022-22-1-long-event-tooltips-no-longer-overflow.md (#3662)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-22-1-long-event-tooltips-no-longer-overflow.md

---
date: ""

> title: Long event tooltips no longer overflow.

Reviewer: ```suggestion
title: Long event or alarm texts no longer overflow in the data point graph tooltip.
```

> title: Long event or alarm texts no longer overflow in the data point graph tooltip.

Reviewer 2: ```suggestion
title: Long event or alarm texts no longer overflow in the data point graph or data explorer tooltips
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64040
version: 1022.22.1
---

> Co-authored-by: Enio Sultan <92023325+eniosultan@users.noreply.github.com>

Reviewer: ```suggestion
In the data point graph or data explorer, long event or alarm texts were overflowing from the tooltip and were not fully visible, making it difficult for users to read the complete event information. This issue has now been fixed. The tooltips for long event or alarm texts are properly displayed without any overflow, allowing users to easily view the entire information. This improvement enhances the usability and readability of alarm and event information in the chart view.
```

> In the data point graph or data explorer, long event or alarm texts were overflowing from the tooltip and were not fully visible, making it difficult for users to read the complete event information. This issue has now been fixed. The tooltips for long event or alarm texts are properly displayed without any overflow, allowing users to easily view the entire information. This improvement enhances the usability and readability of alarm and event information in the chart view.

Reviewer 2: ```suggestion
In the data point graph or data explorer, long event or alarm texts were overflowing from the tooltip and were not fully visible, making it difficult for users to read the complete event information. This issue has now been fixed. The tooltips for long event or alarm texts are properly displayed without any overflow. This improvement enhances the usability and readability of alarm and event information in the chart view.
```
